.github/workflows/android.yml

```diff
@@ -73,8 +73,8 @@ jobs:
         run: | 
           echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
           mkdir upload
-          mv app/build/outputs/apk/debug/app-debug.apk ./upload/omnt-debug.apk
-          mv app/build/outputs/apk/release/app-release.apk ./upload/omnt-release.apk
+          mv app/build/outputs/apk/debug/*debug*.apk ./upload/omnt-debug.apk
+          mv app/build/outputs/apk/release/*release*.apk ./upload/omnt-release.apk
```

Comment:
this one doesn't make sense, but we *could* get rid of renaming the APK again from `CloudCity_OpenMobileNetworkToolkit_debug_<version>` or `CloudCity_OpenMobileNetworkToolkit_release_<version>` into omnt-debug and omnt-release respectivelly.

i think this is a non-issue since it'll only generate one APK per build type, or zero if the build failed.

Reply:
won't add this one. resolving.